server.js

```diff
@@ -15,46 +23,66 @@ app.use(function(req, res, next) {
   next();
 });
 
-/************
- * DATABASE *
- ************/
-
-// var db = require('./models');
-
 /**********
  * ROUTES *
  **********/
 
-// Serve static files from the `/public` directory:
-// i.e. `/images`, `/scripts`, `/styles`
-app.use(express.static('public'));
 
 /*
  * HTML Endpoints
  */
+// provided endpoint
+// app.get('/', function homepage (req, res) {
+//   res.sendFile(__dirname + '/views/index.html');
+// });
+
+// endpoint used in Ski-Me
+app.get('/', function homepage (req, res) {
+  res.sendFile('views/index.html', { root : __dirname });
+});
 
-app.get('/', function homepage(req, res) {
-  res.sendFile(__dirname + '/views/index.html');
+// get all ski runs
+app.get('/api/skiruns', function index(req, res) {
+  db.SkiRun.find({}, function(err, data) {
+    if (err) {
+      console.log(err)
+    }
+    res.json(data);
+  })
 });
 
+// // get one ski run
+app.get('/api/skiruns/:id', function show(req, res) {
+  db.SkiRun.find({_id: req.params.id}, function(err, data) {
+    res.json(data);
+  });
+});
+
+// create new ski run
+// app.post('/api/skiruns', function create(req, res) {
```

Comment:
and of course, you'd need this to be un-commented for the post route to work (this looks reasonable though)